tests/conftest.py

```diff
@@ -22,13 +24,16 @@
 from app.objects.secondclass.c_link import Link
 from app.objects.secondclass.c_fact import Fact
 
+DIR = os.path.dirname(os.path.abspath(__file__))
+CONFIG_DIR = os.path.join(DIR, '..', 'conf')
+
 
 @pytest.fixture(scope='session')
 def init_base_world():
-    with open('conf/default.yml') as c:
+    with open(os.path.join(CONFIG_DIR, 'default.yml')) as c:
         BaseWorld.apply_config('main', yaml.load(c, Loader=yaml.FullLoader))
-    BaseWorld.apply_config('agents', BaseWorld.strip_yml('conf/agents.yml')[0])
-    BaseWorld.apply_config('payloads', BaseWorld.strip_yml('conf/payloads.yml')[0])
+    BaseWorld.apply_config('agents', BaseWorld.strip_yml(os.path.join(CONFIG_DIR, 'agents.yml'))[0])
+    BaseWorld.apply_config('payloads', BaseWorld.strip_yml(os.path.join(CONFIG_DIR, 'payloads.yml'))[0])
```

Comment:
These changes made it easier for me to run my debugger inside pycharm when running specific tests from a module. I probably could have also set up pycharm to declare a specific working directory, but this should fix it for everyone rather than just my pycharm setup.